uint32 dxlID
uint32 address
uint32 value
---
bool txSuccess
